submissions/Levi123/html-css-popup/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class = "menu">
+        <ul class = "navigationList">
+            <li class = "elementList"><a href="#">Mail</a></li>
+            <li class = "elementList"><a href="#">Images</a></li>
+            <li class = "elementList">
+                <input type="checkbox" id="popUp">
+                <label for="popUp" class="popUpBtn">
+                    <img src="icons/popup-button.png" alt = "pop-up menu icon"/>
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

Reply:
Also, Menu button should have `:hover` and `:focus` states

Reply:
for icons left empty attributes alt and add :hover and :focus states